src/RestaurantApi/Dishes/GetDish.js

```diff
@@ -0,0 +1,10 @@
+import { getAllDishes } from './GetAllDishes';
+
+export const getDish = async(id) => {
+  try{
+    const dishesData = await getAllDishes();
+    return dishesData.find(element => element.id === id);
+  } catch (e) {
```

Comment:
no no :D
if we want to fetch single dish we must fetch single dish, no all dishes and find this one. It would be very problematic if we would have 10000 dishes in our database and we want find only specific one.